src/doom/m_menu.c

```diff
@@ -1449,6 +1449,18 @@ boolean M_Responder (event_t* ev)
             key = key_menu_abort;
             joywait = I_GetTime() + 5;
         }
+
+        // Fill-in the savegame name if user press Fire on the joystick
+        if (saveStringEnter && joybmenu >= 0 && ev->data1&1)
+        {
+            // Create a savegame string
+            char savestring[SAVESTRINGSIZE];
+
+            memset(savestring, 0, SAVESTRINGSIZE);
+            M_snprintf(savestring, SAVESTRINGSIZE - 1, "JOYSTICK SLOT %i", saveSlot);
```

Comment:
Why is it necessary to explcitely mention "JOYSTICK" here? I would just call it `SAVEGAME %i`.